point_cloud_transport/include/point_cloud_transport/publisher_plugin.hpp

```diff
@@ -163,17 +160,17 @@ class PublisherPlugin
     const rclcpp::PublisherOptions & options = rclcpp::PublisherOptions())
   {
     advertiseImpl(
-      node_interfaces, base_topic,
+      *node_interfaces.get(), base_topic,
```

Comment:
Do we really want to bypass the reference counter here?

Reply:
https://github.com/ros-perception/point_cloud_transport/pull/129/commits/5a35c97e5ac716276571322499294fbdc068999e